docs/core/stories.rst

```diff
@@ -87,8 +126,8 @@ OR Statements
 -------------
```

Comment:
Make sure all the heading lines are consistent
```suggestion
~~~~~~~~~~~~~
```

Reply:
Checkpoints is supposed to be a larger heading than "Events" in this case (in the same level as  "Format")